### ISSUE TITLE
add "created_at" to follow_requests API endpoint

### ISSUE THREAD

**Issue by mxamber** [mastodon/mastodon#22588]
### Pitch

The `api/v1/follow_requests` endpoint returns a list of accounts, but no information about when the follow request was sent, while the `api/v1/notifications?types[]=["follow_request"]` endpoint returns the date of the follow request, but only of the 30 most recent notifications.

I would appreciate there being some way to get a list of pending follow requests with the time/date the request was made.

### Motivation

This is completely selfish, it's a feature I need. That's the motivation. I'm getting more follow requests than I can go through every day, and I was hoping to automate the process of adding notes to the requesters' profiles containing the date they requested and why.

**Comment by ClearlyClaire** [comment]
Hi! This may be a bit tricky because we currently return accounts rather than follow request objects. That being said, there is precedent to adding fields to `Account` entities on API requests, with `/api/v1/mutes` returning accounts with an additional `mute_expires_at` attribute. We could do something similar, or return a new entity type in some `/api/v2/follow_requests`.

**Comment by imskr** [comment]
I'm new to this project and want to contribute. Can I work on this issue?

**Comment by ClearlyClaire** [comment]
Yes, although I'm not sure what the most appropriate API would be. I personally would be fine with either of my two suggestions above, but other people (e.g. Gargron) may have another view on this.

**Comment by imskr** [comment]
Thanks @ClearlyClaire
